src/nvim/diff.c

```diff
@@ -1598,6 +1598,114 @@ static bool extract_hunk(FILE *fd, diffhunk_T *hunk, diffstyle_T *diffstyle)
   }
 }
 
+static void process_hunk(diff_T **dpp, diff_T **dprevp, int idx_orig, int idx_new, diffhunk_T *hunk,
+                         bool *notset)
```

Comment:
The name of the this variable doesn't make it obvious that it is a pointer. Maybe name it `notsetp` instead.